src/freenas/usr/local/libexec/nas/generate_smb4_conf.py

```diff
@@ -952,9 +952,8 @@ def generate_smb4_conf(client, smb4_conf, role):
     if os.path.exists("/usr/local/etc/smbusers"):
         confset1(smb4_conf, "username map = /usr/local/etc/smbusers")
 
-    server_min_protocol = fs().services.smb.config.server_min_protocol
-    if server_min_protocol != 'NONE':
-        confset2(smb4_conf, "server min protocol = %s", server_min_protocol)
+    if not cifs.enable_smb1:
+        confset2(smb4_conf, "server min protocol = SMB2_02")
```

Comment:
Dont you need to set NT1 in a `else`?

Reply:
Samba defaults to allow NT1, but I agree it's better to explicitly set it. Will make that change.